.buildkite/pipeline.yml

```diff
@@ -14,6 +14,13 @@ steps:
         Pkg.add("CUDA")
         Pkg.add("KernelAbstractions")
         Pkg.add("NLPModels")
+        Pkg.add("JuMP")
+        Pkg.add("ForwardDiff")
+        Pkg.add("NLPModelsIpopt")
+        Pkg.add("NLPModelsJuMP")
+        Pkg.add("Percival")
+        Pkg.add("PowerModels")
+        Pkg.add("MadNLP")
```

Comment:
not sure if this is ideal. Maybe we can create a separate Project.toml file for testing?

Reply:
The best solution is to create one `Project.toml` for each dependency. We can create a CI build for each extension if we do that.

Reply:
Can't we do, say, `Project.toml` on `/test` and have `Pkg.add("CUDA.jl")` and `Pkg.add("KernelAbstractions.jl")` before project environment instantiation?

Reply:
We can do that, it's what I do for KrylovPreconditioners.jl:
https://github.com/JuliaSmoothOptimizers/KrylovPreconditioners.jl/blob/main/.buildkite/pipeline.yml

Reply:
I think this approach is good. Can we do it in this way?